lib/src/index.js

```diff
@@ -11,16 +11,22 @@ var isValidGlob = require('is-valid-glob');
 
 var getContents = require('./get-contents');
 var wrapWithVinylFile = require('./wrap-with-vinyl-file');
+var normalizeOption = require('../normalize-option');
 
 function src(glob, opt) {
-  var options = assign({
-    read: true,
-    buffer: true,
-    stripBOM: true,
-    sourcemaps: false,
-    passthrough: false,
-    followSymlinks: true,
-  }, opt);
+  if (!opt) {
+    opt = {};
+  }
+
+  var options = assign({}, opt, {
+    buffer: normalizeOption('boolean', true, opt.buffer, glob),
```

Comment:
not sure how I'm feeling about the glob being passed in now that I see it.  I don't think it is useful to determine the options.  I'm going to think a bit more on a way to defer option resolution until we have a Vinyl File object.